test/js/ui/map.test.js

```diff
@@ -103,7 +98,7 @@ test('Map', function(t) {
             style.fire('source.error');
             style.fire('source.change');
             style.fire('tile.add');
-            style.fire('tile.load');
+            // style.fire('tile.load');
```

Comment:
:+1:, removing the comments, opened https://github.com/mapbox/mapbox-gl-js/issues/2360